### PR TITLE
[WFLY-11789]: Intermittent failures in NotClosingInjectedContextTestCase.

### PR DESCRIPTION
**Pull request by ehsavoie**
Ensuring that  we get the destination before sending to it.

Jira: https://issues.jboss.org/browse/WFLY-11789

### CODE REVIEW

**Comment by ehsavoie** [comment]
retest this please

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/messaging/jms/context/notClosingInjectedContext/auxiliary/Ejb.java [hunk 1]
```diff
@@ -55,7 +57,12 @@ public void send(String text) {
     private void send(String text, JMSContext jmsContext) {
         try {
             LOGGER.info("Sending: " + text);
-            jmsContext.createProducer().send(lookup(Mdb.JNDI_NAME), text);
+            long start = System.currentTimeMillis();
+            Queue queue = lookup(Mdb.JNDI_NAME);
+            while(queue == null && (System.currentTimeMillis() - start < TIMEOUT)) {
+                queue = lookup(Mdb.JNDI_NAME);
```
Comment (kabir) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/messaging/jms/context/notClosingInjectedContext/auxiliary/Ejb.java]: @ehsavoie I'd add a little sleep after the lookup, say 200ms

Comment (ehsavoie) [reply testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/messaging/jms/context/notClosingInjectedContext/auxiliary/Ejb.java]: I agree :)

**Comment by bstansberry** [comment]
@ehsavoie 

[05:37:23][Step 2/3] [ERROR] /store/work/tc-work/bb15431f347cd651/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/messaging/jms/context/notClosingInjectedContext/auxiliary/Ejb.java:25:8: Unused import - java.util.logging.Level. [UnusedImports]
[05:37:23][Step 2/3] Audit done.
[05:37:23][Step 2/3] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.0.0:checkstyle (check-style) on project wildfly-ts-integ-basic: An error has occurred in Checkstyle report generation.

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
@ehsavoie Needs a rebase.

**Pull request merged**